jobdsl/organisations.groovy

```diff
@@ -14,7 +14,8 @@ List<Map> orgs = [
         [name: 'FinRem', displayName: 'Financial Remedy', regex: 'finrem.*'],
         [name: 'CDM', regex: '\\b(?:document-management-store-app|dm-shared-infrastructure|ccd.*)\\b'],
         [name: 'IAC', regex: 'ia.*'],
-        [name: 'Platform', regex: '(rpe-.*|send-letter.*|draft-store.*|bulk-scan.*|cmc-pdf-service|feature-toggle.*|private-beta-invitation.*|service-auth-provider-app|spring-boot-template|data-extractor|camunda-.*)'],
+        [name: 'BSP', regex: '(send-letter-service|send-letter-performance-tests|bulk-scan.*)'],
```

Comment:
Few other configs here, if you want to change : https://github.com/hmcts/cnp-jenkins-config/blob/master/team-config.yml#L104

Reply:
applied changes. didn't touch namespace though